src/bp/core/routers/admin/users.ts

```diff
@@ -8,7 +7,13 @@ import _ from 'lodash'
 
 import { CustomRouter } from '../customRouter'
 import { ConflictError } from '../errors'
-import { assertBotpressPro, needPermissions, success as sendSuccess, validateBodySchema } from '../util'
+import {
```

Comment:
That's a question I had since day 1 and never got satisfied. I thought of creating an object containing both (actually I already did, it's UniqueUser, with both properties), but I wanted to have a fair discussion before changing everything. I've separated them (strategy vs user) at first while implementing different blocks of code.

But I wouldn't create a hash of both, since sometimes you need to identify the strategy for specific rules